src/ansi-c/ansi_c_language.h

```diff
@@ -22,13 +22,30 @@ Author: Daniel Kroening, kroening@kroening.com
 // clang-format off
 #define OPT_ANSI_C_LANGUAGE \
   "(max-nondet-tree-depth):" \
-  "(min-null-tree-depth):"
+  "(min-null-tree-depth):" \
+  "(pointers-to-treat-as-array):" \
+  "(associated-array-sizes):" \
+  "(max-dynamic-array-size):" \
+  "(pointers-to-treat-as-string):"
```

Comment:
`pointers-to-treat-as-string` -> `pointers-to-treat-as-strings` (and throughout the PR)